src/export.js

```diff
@@ -1,51 +1,42 @@
-// Deprecated
-// Extend assert methods to QUnit for Backwards compatibility
-( function() {
-	var i,
-		assertions = Assert.prototype;
+function applyDeprecated( name ) {
+	return function() {
+		throw new Error(
+			"`" + name + "` is removed in QUnit 2.0, you should now use the " +
```

Comment:
ef9c175 what do you think? Also the global message is `"The global `ok` is removed..."`